templates/resources/domain_zone_dynhost_record.md.tmpl

```diff
@@ -0,0 +1,28 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "ovh_domain_zone_dynhost_record Resource - terraform-provider-ovh"
+subcategory: ""
+description: |-
+  
+---
+
+# ovh_domain_zone_dynhost_record (Resource)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `zone_name` (String) Zone name
+
+### Optional
```

Comment:
the templates and docs will need to be re-generated according the the fixes to do in resources' schemas